pkg/kubernetes/tool.html

```diff
@@ -254,5 +328,151 @@ <h4 class="modal-title">Pod Editor</h4>
     </div>
   </div>
 
+  <div id="pod-delete" class="modal" tabindex="-1" role="dialog">
```

Comment:
I would suggest using a single dialog for confirmation of both pod and minion deletion. This is an exact copy paste of the following element.